subiquity/models/filesystem.py

```diff
@@ -356,9 +482,15 @@ def _use_disk(self, disk):
         if disk.path not in self._disks:
             self._disks[disk.path] = disk
 
+    def all_mounts(self):
+        return self._mounts[:]
```

Comment:
I've not seen this notation; what does it do ver returning the object name?

Reply:
It means make a copy of the list as protection against the caller mutating the returned value. Probably not necessary but for the avoidance of surprises...